aslam_cv_cameras/test/test-distortions.cc

```diff
@@ -99,6 +99,31 @@ TYPED_TEST(TestDistortions, DistortAndUndistortUsingExternalParameters) {
   }
 }
 
+TYPED_TEST(TestDistortions, TemplatedDistortUsingExternalParameters) {
```

Comment:
I'm not sure if aslam has a dependency to ceres but if so would it make sense to add an autodiff vs. symbolic unit test?